lib/xcodeproj/workspace.rb

```diff
@@ -82,6 +82,14 @@ def self.new_from_xcworkspace(path)
     def self.from_s(xml, workspace_path = '')
       document = REXML::Document.new(xml)
       instance = new(document)
+      # Load projects reference in "Pods/" if needed
+      pods_path = "#{File.expand_path("..", workspace_path)}/Pods"
```

Comment:
I think this project doesnt know about "Pods". This fix might be better in CocoaPods gem instead.